### PR TITLE
 TIP-856: Move Pim\Batch Component/Bundle to Akeneo\Tool 

### PR DESCRIPTION
**Pull request by arnolanglade**
A new name space has been created in this PR `Akeneo\Tool` which will contains our tools. All components and bundles from `Akeneo\Xxx` will be move to `Akeneo\Tool\Xxx`.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | yes
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: CHANGELOG-3.0.md [hunk 1]
```diff
@@ -54,3 +54,5 @@
 
 - Move namespace `Pim\Component\Api` to `Akeneo\Tool\Component\Api`
 - Move namespace  `Pim\Bundle\ApiBundle` to `Akeneo\Tool\Bundle\ApiBundle`
+- Move namespace `Pim\Component\Batch` to `Akeneo\Tool\Component\Batch`
+- Move namespace  `Pim\Bundle\BatchBundle` to `Akeneo\Tool\Bundle\BatchBundle`
```
Comment (pierallard) [inline CHANGELOG-3.0.md]: douple space (and 2 lines before too)

File: src/Akeneo/Tool/Bundle/BatchBundle/Resources/config/entities.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+parameters:
+    akeneo_batch.entity.warning.class:        Akeneo\Tool\Component\Batch\Model\Warning
+    akeneo_batch.entity.job_instance.class:   Akeneo\Tool\Component\Batch\Model\JobInstance
+    akeneo_batch.entity.job_execution.class:  Akeneo\Tool\Component\Batch\Model\JobExecution
+    akeneo_batch.entity.step_execution.class: Akeneo\Tool\Component\Batch\Model\StepExecution
+    akeneo_batch.job.batch_status.class:      Akeneo\Tool\Component\Batch\Job\BatchStatus
+    akeneo_batch.job.job_parameters.class:    Akeneo\Tool\Component\Batch\Job\JobParameters
```
Comment (pierallard) [inline src/Akeneo/Tool/Bundle/BatchBundle/Resources/config/entities.yml]: I hope it's a git mv but not really well displayed by github...

File: src/Akeneo/Tool/Bundle/BatchBundle/Resources/config/jobs.yml [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+    akeneo_batch.job_repository.class:                      Akeneo\Tool\Bundle\BatchBundle\Job\DoctrineJobRepository
+    akeneo_batch.job.job_instance_repository.class:         Akeneo\Tool\Bundle\BatchBundle\Job\JobInstanceRepository
+    akeneo_batch.job.job_registry.class:                                           Akeneo\Tool\Component\Batch\Job\JobRegistry
+    akeneo_batch.job.job_parameters.default_values_provider_registry.class:        Akeneo\Tool\Component\Batch\Job\JobParameters\DefaultValuesProviderRegistry
+    akeneo_batch.job.job_parameters.constraint_collection_provider_registry.class: Akeneo\Tool\Component\Batch\Job\JobParameters\ConstraintCollectionProviderRegistry
+    akeneo_batch.job.job_parameters.empty_values_provider.class:                   Akeneo\Tool\Component\Batch\Job\JobParameters\EmptyDefaultValuesProvider
+    akeneo_batch.job.job_parameters.empty_constraint_collection_provider.class:    Akeneo\Tool\Component\Batch\Job\JobParameters\EmptyConstraintCollectionProvider
+    akeneo_batch.job.job_parameters_validator.class:                               Akeneo\Tool\Component\Batch\Job\JobParametersValidator
```
Comment (pierallard) [inline src/Akeneo/Tool/Bundle/BatchBundle/Resources/config/jobs.yml]: Is it normal to keep the old name ?

File: CHANGELOG-3.0.md (hunk 1 above)
Comment (arnolanglade) [reply CHANGELOG-3.0.md]: it is ok, it is a git mv!

File: src/Akeneo/Tool/Bundle/BatchBundle/Resources/config/jobs.yml (hunk 3 above)
Comment (arnolanglade) [reply src/Akeneo/Tool/Bundle/BatchBundle/Resources/config/jobs.yml]: it is ok, it is a git mv!

**Pull request merged**